src/gui/windows/creative_inventory.zig

```diff
@@ -18,13 +20,18 @@ pub var window = GuiWindow{
 		.{.attachedToFrame = .{.selfAttachmentPoint = .lower, .otherAttachmentPoint = .lower}},
 		.{.attachedToFrame = .{.selfAttachmentPoint = .middle, .otherAttachmentPoint = .middle}},
 	},
-	.contentSize = Vec2f{64*8, 64*4},
+	.contentSize = Vec2f{64*8, 64*6},
 	.scale = 0.75,
 };
 
 const padding: f32 = 8;
+const slotsPerRow: u32 = 10;
 var items: main.List(Item) = undefined;
 var inventory: Inventory = undefined;
+var searchInput: *TextInput = undefined;
+var searchString: []const u8 = undefined;
+var searchSelectionStart: ?u32 = null;
+var searchCursor: ?u32 = null;
```

Comment:
Instead of doing this with global variables, accessed from three different functions, better just do it all locally in the filter function.